Demo/Common/IntegrationExamples/Components/BLIKComponentAdvancedFlowExample.swift

```diff
@@ -0,0 +1,145 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import AdyenActions
+import AdyenCheckout
+import AdyenComponents
+
+internal final class BLIKComponentAdvancedFlowExample: InitialDataAdvancedFlowProtocol {
+
+    internal weak var presenter: PresenterExampleProtocol?
+    
+    private var adyenCheckout: AdyenCheckout?
+    private var component: AdyenCheckoutComponent?
+    
+    internal lazy var apiClient = ApiClientHelper.generateApiClient()
+    
+    // comes from demo app protocol, unused on new structure
+    internal lazy var context: AdyenContext = generateContext()
+    
+    internal init() {}
+    
+    internal func start() {
+        presenter?.showLoadingIndicator()
+        requestPaymentMethods(order: nil) { [weak self] result in
```

Comment:
The `order` still doesn't sit well with me as because of Uplift order will mean something different in a year or so. I don't have a better name now :)